ckanext/honduras/templates/footer.html

```diff
@@ -0,0 +1,63 @@
+<footer class="site-footer">
+    <div class="container">
+      {% block footer_content %}
+      <div class="row">
+        <div class="col-md-4">
+            <img src="images/honduras-white-logo.png" width="180" class="logo"/>
```

Comment:
You need an extra slash at the beginning or the logo won't load on pages other than the homepage

```suggestion
            <img src="/images/honduras-white-logo.png" width="180" class="logo"/>
```